config/routes.rb

```diff
@@ -21,15 +21,15 @@
   get 'images.xml', to: 'images#images'
 
   controller :package do
-    get 'package/:package' => :show, :constraints => { :package => /[-+\w\.:\@]+/ }
-    get 'package/thumbnail/:package.png' => :thumbnail, :constraints => { :package => /[-+\w\.:\@]+/ }
-    get 'package/screenshot/:package.png' => :screenshot, :constraints => { :package => /[-+\w\.:\@]+/ }
+    get 'package/:package', action: :show, constraints: { package: /[-+~\w\.:\@]+/ }
```

Comment:
Rubocop has a `Style/HashSyntax` cop, which can correct that for you :wink:  Rubocop doesn't complain because the config directory is excluded in [.rubocop.yml](https://github.com/openSUSE/software-o-o/blob/master/.rubocop.yml#L10)